django/thunderstore/api/cyberstorm/views/team.py

```diff
@@ -52,6 +62,58 @@ def get_queryset(self) -> QuerySet[TeamMember]:
         )
 
 
+class CyberstormTeamAddMemberRequestSerialiazer(serializers.Serializer):
+    user = serializers.CharField(max_length=User._meta.get_field("username").max_length)
+    role = serializers.ChoiceField(choices=TeamMember._meta.get_field("role").choices)
+
+
+class CyberstormTeamAddMemberResponseSerialiazer(serializers.Serializer):
+    user = serializers.CharField(max_length=User._meta.get_field("username").max_length)
+    role = serializers.ChoiceField(choices=TeamMember._meta.get_field("role").choices)
+    team = serializers.CharField(max_length=Team._meta.get_field("name").max_length)
+
+
+class AddTeamMemberAPIView(APIView):
+    @swagger_auto_schema(
+        request_body=CyberstormTeamAddMemberRequestSerialiazer,
+        responses={200: CyberstormTeamAddMemberResponseSerialiazer},
+        operation_id="cyberstorm.team.members.add",
+        tags=["cyberstorm"],
+    )
+    def post(self, request, team_name, format=None):
+        try:
+            team = Team.objects.get(name=team_name)
+        except Team.DoesNotExist:
+            return Response(
+                json.dumps(
+                    {
+                        "error": "Team not found",
+                    }
+                ),
+                status=status.HTTP_400_BAD_REQUEST,
+            )
```

Comment:
Generally if you need to use `json.dumps` in API code, you're _probably_ doing something wrong. DRF comes bundled in with a perfectly good `NotFound` exception which you can raise in situations like this, read more on the exceptions DRF provides by default here: https://www.django-rest-framework.org/api-guide/exceptions/?q=get_object_or_404#notfound

That said, in this case you probably want to do the following:
```python
from rest_framework.generics import get_object_or_404

team = get_object_or_404(Team, name=team_name)
```